src/pudl/metadata/warnings.py

```diff
@@ -19,5 +19,9 @@
     "scale_hazard": "Extremely large table; do not attempt to open with Excel.",  # TODO: set a threshold
     "outliers": "Outliers present.",
     "missing_years": "Some years are missing from the data record.",
-    "ferc_is_hard": "FERC data is notoriously difficult to extract cleanly; see TODO for details.",
+    "ferc_is_hard": (
+        "FERC data is notoriously difficult to extract cleanly; see "
+        "`Notable Irregularities <https://catalystcoop-pudl.readthedocs.io/en/latest/data_sources/ferc1.html#notable-irregularities>`_ "
```

Comment:
I hard-coded in the published URL but could see an argument for figuring out an rst `:ref:` instead; lmk if that's preferable

Reply:
eep i also added a brief description here in the ferc1 pr.. but i like the idea of linking to the docs.

and i'll be honest i don't know if :ref: or straight up link would be preferable.